pkg/client/results/processing.go

```diff
@@ -130,6 +132,49 @@ func (i *Item) GetSubTreeByName(root string) *Item {
 	return nil
 }
 
+// manualResultsAggregation is custom logic just for aggregating results for the top level summary
```

Comment:
Will need some good tests for this; I have some but I think it needs changes/tests to cover the daemonset case.

Not only will the top-summary need its status, but the objects right below the top level summary are the 'node summary' so we need to make sure that we arent leaving one layer blank of status values.